packages/extension/src/shared/token/utils.ts

```diff
@@ -17,21 +17,18 @@ export const alphTokens: Token[] = defaultNetworkIds.map((networkId) => {
   }
 })
 
-export const tokensFromAlephiumTokenList: Token[] = alphTokens.concat(
-  [mainnetTokensMetadata, testnetTokensMetadata].flatMap(convertTokenList)
-)
-
 export const dustALPHAmount = BigInt(1000000000000000)
 
 export const minimumALPHAmount = (tokenNums: number) => {
   return dustALPHAmount + BigInt(tokenNums * 100 * 1000000000)
 }
 
-function convertTokenList(tokenList: TokenList): Token[] {
+export function convertTokenList(tokenList: TokenList): TokenListToken[] {
+  const now = new Date().getTime()
   return tokenList.tokens.flatMap((token) => {
     const networkId = defaultNetworkIds[tokenList.networkId]
     if (networkId) {
-      return [{ networkId, verified: true, ...token }]
+      return [{ networkId, verified: true, updatedAt: now, ...token }]
```

Comment:
This uses too much space since all tokens have to store the same piece of information.

Same for `verified: true`. Do we still need this verified field?

Reply:
Thanks for the feedback! Now the `verified` is removed and `updatedAt` is only stored once.

Reply:
No need to migrate I think, we could just remove the field from the code.

Reply:
Token from full node is not verified and will have a warning badge displayed. I am making it a bit more clear [here](https://github.com/alephium/extension-wallet/blob/auto-update-token-list/packages/extension/src/ui/features/accountTokens/tokens.state.ts#L263) now. WDTY?

Reply:
LGTM. I thought all of tokens in `tokenStore` are verified.

Question: if a token is `verified: false` in `tokenStore` and later it's added into `tokenListStore` after token-list update, 
1. will the wallet show it as verified?
2. will the token be stored in both `tokenListStore` and `tokenStore`?

Reply:
Great questions. 
If a token was in `tokenStore` and was added to the token list, it will be removed from the `tokenStore`. The warning badge will be removed.

Reply:
Great. `useTokensInNetwork` is not very efficient, it should be something like:
```
return useMemo(() => computeTokens(), [tokenListTokens, tokens])
```

Reply:
Indeed, fixed. Thanks!